src/icloudpd/authentication.py

```diff
@@ -87,7 +110,7 @@ def request_2sa(icloud: pyicloud_ipd.PyiCloudService, logger: logging.Logger):
 
     code = click.prompt("Please enter two-factor authentication code")
     if not icloud.validate_verification_code(device, code):
-        logger.error("Failed to verify two-factor authentication code")
+        logger.error(f"Failed to verify two-factor authentication code for device \"{device.get('deviceName')}\"")
```

Comment:
I'll remove this extra device info--I added for debugging, but it wasn't helpful because it was returning `None`.